src/components/Books.js

```diff
@@ -5,40 +5,37 @@ import PropTypes from 'prop-types';
 import { deleteBook, removebook } from '../redux/books/booksSlice';
 import 'react-circular-progressbar/dist/styles.css';
 
-function Books({
-  id, title, author,
-}) {
+function Books({ id, title, author }) {
   const dispatch = useDispatch();
   const handleDelete = async () => {
     dispatch(removebook({ id }));
     await dispatch(deleteBook({ id }));
   };
   return (
     <>
-      <div className="books-container">
-        <div className="book-item">
-          <div className="section1">
+      <div className="books__container">
+        <div className="book__item">
+          <div className="section__one">
             <h2>{title}</h2>
             <p>{author}</p>
-            <div className="book-button-section">
+            <div className="book__btn__section">
               <button type="button">Comments</button>
-              <button onClick={handleDelete} type="button">Remove</button>
+              <button onClick={handleDelete} type="button">
+                Remove
+              </button>
               <button type="button">Edit</button>
             </div>
           </div>
-          <div className="section2">
-            <div className="progress-bar">
-              <CircularProgressbar value="75" text="75%" />
+          <div className="section__two">
+            <div className="progress__bar">
+              <CircularProgressbar value="65" text="65%" />
             </div>
-            <div>
-              <h2>
-                75
-                %
-              </h2>
+            <div className="progres__text">
+              <h2>65 %</h2>
               <p>Completed</p>
             </div>
           </div>
-          <div className="section3">
+          <div className="section__three">
             <p>CURRENT CHAPTER</p>
             <h3>Chapter 17</h3>
             <button type="button">Update Progress</button>
```

Comment:
- [ Required ] Good job styling the application, however, I've noted some differences between your design and the design on Zeplin. Please see the list and the screenshot below for details on this.

  ## Navigation menu
  - [x] The background color of the navigation menu on the Zeplin design is `#ffffff` in color whereas the navigation menu on your design has a `#fafafa` background color.
  
  - [x] The Zeplin design has a user icon on the top right section of the navigation menu which has the shape of a `circle` whereas your design has a user icon that is `oval` in shape.
 
  - [x] The Zeplin design has a  grayish `border-bottom` on the navigation menu whereas your design has no clear border that distinguishes it from the body of the booklist.
  
  - [x] The text on the navigation menu `Books` and `Categories` should be left aligned, currently, the text on the menu is centered using the flex box property `space-between` please use `flex-start` to align them to the left.
  
  ## Add book form
  - [x] The alignment of the components in the Zeplin design has the same margin or padding that makes them appear on the same line both on the `left` and the `right`. The `add book` form's submit button should be right aligned and be on the same line as the right edge of the `BookList` and `User icon`
  
  ## Book list
  - [x] The following text `Book title`, `Book author` and `Comments` should be left aligned and aligned on the same line. Currently, the `Comment` text has a `padding` of `20px` which makes it not to be on the same line as the `Book title` and `Book author` text.
  

## Zeplin design 
![Screenshot from 2023-06-17 17-35-15](https://github.com/musangamfure/bookstore/assets/16392046/996663a5-a290-4964-bf00-e34500e6871d)


## Your design
![Screenshot from 2023-06-17 17-22-25](https://github.com/musangamfure/bookstore/assets/16392046/223f632a-ec71-46db-9579-75c565dcbafa)